web/src/main/kotlin/com/example/ajaxproject/model/PrivateChatMessage.kt

```diff
@@ -8,7 +8,7 @@ import org.springframework.data.mongodb.core.mapping.Document
 data class PrivateChatMessage(
     @Id
     val id: ObjectId,
-    val privateChatRoom: PrivateChatRoom,
+    val privateChatRoom: String,
```

Comment:
```suggestion
    val privateChatRoomId: String,
```
?

Reply:
Yes , i agree that its better to name it as a field  for id